tests/test_fragments.py

```diff
@@ -5,6 +5,32 @@
 from gator import fragments
 
 
+@pytest.mark.parametrize(
+    "writing_string,expected_count",
+    [
+    ("Paragraph1\n - first item\n - second item\n\nParagraph2", 2),
+    ("Paragraph1\n 1. item one\n 2. item two\n\nParagraph2", 2),
+    (" - first item\n - second item\n", 0),
+    ]
+)
+def test_listsbreak_paragraphs(writing_string, expected_count):
```

Comment:
Hi @lylet-AC, instead of adding a new test method, could you add your cases to the correct existing test method?